include/sway/server.h

```diff
@@ -95,6 +95,7 @@ struct sway_server {
 		struct wlr_session_lock_manager_v1 *manager;
 
 		struct wlr_session_lock_v1 *lock;
+		struct wlr_surface *focused;
```

Comment:
Instead of having this new field, can we use `wlr_seat.keyboard_state.focused_surface`?

Reply:
Not as easily - that field is overwritten by `swaymsg focus` and a number of other events, so it would need to be saved and restored to avoid the focus wandering to non-lockscreen surfaces.

Reply:
Hm, I see. Let's go with this then.